ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -145,6 +146,29 @@ level_can_be_refined(int level_number, int max_levels)
     return level_number < finest_level_number;
 }
 
+static inline std::pair<int, int>
```

Comment:
```suggestion
inline std::pair<int, int>
```
would you please add a docstring?

Reply:
Why should we remove the `static` keyword?

Reply:
static here means that the function will only be available in the current translation unit, which doesn't make any sense when it is declared and defined in a header.